### PR TITLE
Fixes Blog link, Removes unused PrimaryNav component and config

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* https://oktainc.atlassian.net/browse/BD-635

### CODE REVIEW

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 1]
```diff
@@ -105,39 +105,6 @@ module.exports = ctx => ({
...
-          { text: 'Authentication', link: '/product/authentication/' },
-          { text: 'Authorization', link: '/product/authorization/' },
-          { text: 'User Management', link: '/product/user-management/' },
-          { type: 'divider' },
-          { text: 'Publish an integration' },
-          { text: 'Overview', link: '/okta-integration-network/' },
-        ]
-      },
-      { text: 'Pricing', link: 'https://www.okta.com/pricing/#customer-identity-products' },
-      { text: 'Blog', link: 'https://developer.okta.com/blog/' },
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:133]: @albertchen-okta this is confusing, there is this old set of configs that aren't used anymore, so fixing this one didn't fix the blog link on the live site

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 2]
```diff
@@ -224,7 +191,8 @@ module.exports = ctx => ({
       { text: 'Guides', link: '/docs/guides/' },
       { text: 'Concepts', link: '/docs/concepts/' },
       { text: 'Reference', link: '/docs/reference/' },
-      { text: 'Languages & SDKs', link: '/code/' }
+      { text: 'Languages & SDKs', link: '/code/' },
+      { text: 'Release Notes', link: '/docs/release-notes/' }
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:195]: Updated so the mobile nav matches the desktop left nav

File: packages/@okta/vuepress-site/.vuepress/config.js (hunk 1 above)
Comment (albertchen-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:133]: ah no wonder, thanks

File: packages/@okta/vuepress-site/.vuepress/config.js (hunk 2 above)
Comment (albertchen-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:195]: thanks!

**Review by albertchen-okta — APPROVED** [review]

File: packages/@okta/vuepress-theme-prose/components/PrimaryNav.vue [hunk 3]
```diff
@@ -1,50 +0,0 @@
-<template>
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/components/PrimaryNav.vue:1]: Is this component not being used by the "new" nav?

Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/components/PrimaryNav.vue:1]: I couldn't find any usages of the component in the project, it's all just in `Header.vue`

**Pull request merged**